workflows/phylorthology.nf

```diff
@@ -262,7 +276,7 @@ workflow PHYLORTHOLOGY {
     // All outputs are needed for species tree inference, but not for the
     // remainder.
     SPECIES_TREE_PREP(
-        ch_core_gene_trees.collect(),
+        ch_core_gene_trees,
```

Comment:
it's been a while since i looked at these, are you sure about this?

Reply:
Yup!